shared/css/conversation.css

```diff
@@ -416,6 +468,8 @@ html[dir="rtl"] .settings-menu.dropdown-menu {
 .remote > .avatar {
   /* make visually distinct from local avatar */
   opacity: 0.25;
+  background-image: url("../img/local-avatar-thumb.svg");
```

Comment:
This appears wrong, you're saying `.avatar` uses local, then `.remote .avatar` uses guest, then finally `.remote > .avatar` uses local.

If you change these three like I suggested above, then you'd end up with just two: `.avatar` with local, and `.avatar.remote` with guest.